SConstruct

```diff
@@ -586,25 +611,127 @@ def list_sources(base, suffixes):
                     yield os.path.normpath(path)
     return list(_iter(base))
 
-def add_soci_sources(env, object_builder, is_unity):
+def append_sources (result, *filenames, **kwds):
+    result.append([filenames, kwds])
+
+def get_soci_sources(style):
+    result = []
     cpp_path = [
         'src/soci/src/core',
         'src/sqlite',]
-    object_builder.add_source_files(
+    append_sources(result,
```

Comment:
Hmm, I think this isn't valid indentation still!  :-)

Sorry to be a stickler, but Python is almost unique inasmuch it has an "official" style guideline endorsed by the BDFL.